marquez_client/marquez.py

```diff
@@ -8,35 +20,37 @@
 
 class MarquezClient(object):
     API_PATH = "api/v1"
-    MARQUEZ_HOST_KEY = "MQZ_HOST"
-    MARQUEZ_PORT_KEY = "MQZ_PORT"
+    DEFAULT_TIMEOUT = 5
```

Comment:
This is in seconds, but for the uninitiated reader of this code, that's not obvious. Maybe we can rename this to `DEFAULT_TIMEOUT_SECS` or something similar?

Reply:
ok, sounds good.